ansible/roles/candlepin-user/tasks/checkout.yml

```diff
@@ -5,13 +5,13 @@
     dest: "{{candlepin_checkout}}"
     force: yes
     version: "{{candlepin_branch}}"
-    refspec: '+refs/pull/*:refs/heads/*'
-  when: "{{candlepin_git_pull}}"
+#    refspec: '+refs/pull/*:refs/heads/*'
```

Comment:
It should be removed, because git checkout of candlepin PR is failling with master branch of caracalla: look at this test result: https://rhsm-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/candlepin/job/CandlepinPerformance/693/console